zhaquirks/__init__.py

```diff
@@ -33,6 +33,25 @@ def __init__(self, *args, **kwargs):
 class LocalDataCluster(CustomCluster):
     """Cluster meant to prevent remote calls."""
 
+    async def bind(self):
+        """Bind cluster."""
+        return (foundation.Status.SUCCESS,)
+
+    async def unbind(self):
+        """Unbind cluster."""
+        return (foundation.Status.SUCCESS,)
+
+    async def configure_reporting(
+        self,
+        attribute,
+        min_interval,
+        max_interval,
+        reportable_change,
+        manufacturer=None,
+    ):
```

Comment:
And it is better to override the actual ZCL command which is _configure_reporting() method, as soon we could configure multiple attrs at once for reporting, eg https://github.com/home-assistant/core/blob/9907e95c34d0763838756e06497cc0bf905d78e4/homeassistant/components/zha/core/channels/hvac.py#L317

```suggestion
    async def configure_reporting(
        self,
        attribute,
        min_interval,
        max_interval,
        reportable_change,
        manufacturer=None,
    ):
```

Reply:
Done